packages/peregrine/lib/hooks/useSort.js

```diff
@@ -2,9 +2,9 @@ import { useState } from 'react';
 
 const defaultSort = {
     sortText: 'Best Match',
-    sortId: 'sortItem.relevance',
-    sortAttribute: 'relevance',
-    sortDirection: 'DESC'
+    sortId: 'sortItem.position',
+    sortAttribute: 'position',
+    sortDirection: 'ASC'
```

Comment:
This makes the default `position` but when you click the `Sort` dropdown there is no selection matching it. And then if you click "Best Match" it sets it back to `relevance`. Perhaps we should add an item in the [default props sort array](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/ProductSort/productSort.js#L114-L135). Should the `sortText` be "Position" then?

Reply:
It was a minor change so I went ahead and did this :)